tox.ini

```diff
@@ -8,22 +8,8 @@ deps =
     django20: Django>=2.0,<2.1
     django21: Django>=2.1,<2.2
     django22: Django>=2.2,<2.3
-    -r{toxinidir}/requirements/django.txt
-commands = pytest --disable-pytest-warnings --nomigrations {posargs}
-
-
-[testenv:pytest-py27]
-basepython = python2.7
-deps =
     -r{toxinidir}/requirements/test.txt
-commands = pytest --disable-pytest-warnings --ignore=opaque_keys/edx/django {posargs}
-
-
-[testenv:pytest-py35]
-basepython = python3.5
-deps =
-    -r{toxinidir}/requirements/test.txt
-commands = pytest --disable-pytest-warnings --ignore=opaque_keys/edx/django {posargs}
```

Comment:
I can't say I get all this, but if tests are still passing and they are running under the right environment, seems good to me.